test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1

```diff
@@ -73,9 +79,19 @@ finally {{
             try {
                 Wait-UntilTrue { (Get-PSHostProcessInfo -Id $powershell.Id) -ne $null }
 
-                "Enter-PSHostProcess -Id $($powershell.Id) -ErrorAction Stop
-                `$pid
-                Exit-PSHostProcess" | pwsh -c - | Should -Be $powershell.Id
+@'
+try {{
+    Enter-PSHostProcess -Id {0} -ErrorAction Stop
+    $pid
+}}
+catch {{
+    $_
+}}
+finally {{
+    Exit-PSHostProcess
+}}
```

Comment:
double braces?

Reply:
You need this for template strings to escape